src/config/disabledOpts.ts

```diff
@@ -27,7 +27,7 @@ export const disabledOpts: DisabledOptions = {
 	},
 	paysWithFeeOrigin: {
 		disabled: true,
```

Comment:
Shouldn't this just be `disabled: false`

Reply:
Yup nice catch.

Reply:
fixed